src/configuration/currencies.ts

```diff
@@ -1,12 +1,11 @@
 import coininfo from "coininfo";
 
-export enum AddressType {
+export enum DerivationMode {
   LEGACY = "Legacy",
   NATIVE = "Native SegWit",
-  SEGWIT = "SegWit",
+  SEGWIT = "Segwit",
```

Comment:
`SegWit` -> `Segwit` intended? feels more consistent to keep `SegWit` (especially because for `NATIVE` the value is `Native SegWit`)

Reply:
You are correct. This is a mistake.